opensrp-chw-referral/src/main/java/org/smartregister/chw/referral/util/CopyToClipboardDialog.java

```diff
@@ -0,0 +1,60 @@
+package org.smartregister.chw.referral.util;
+
+import android.app.Dialog;
+import android.content.ClipData;
+import android.content.ClipboardManager;
+import android.content.Context;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.VisibleForTesting;
+import android.view.View;
+import android.view.Window;
+import android.widget.TextView;
+
+
+import org.smartregister.chw.referral.R;
+
+import timber.log.Timber;
+
+public class CopyToClipboardDialog extends Dialog implements View.OnClickListener {
```

Comment:
There is a class with similar name available on **chw-core** module that you might consider extending instead

Reply:
@ellykits chw-referral module is used as a dependency to chw-core, adding chw-core to the module would lead to cyclic dependency between the 2 modules

Reply:
Alright. I thought it was the other way round. Makes sense.